diff --git a/src/Form/EntityBreakLockForm.php b/src/Form/EntityBreakLockForm.php
index 3e36ceb..ac6cea9 100644
--- a/src/Form/EntityBreakLockForm.php
+++ b/src/Form/EntityBreakLockForm.php
@@ -90,8 +90,7 @@ class EntityBreakLockForm extends FormBase {
       $form_state->setRedirectUrl($url);
     }
     else {
-      $language = $this->languageManager->getLanguage($form_state->get('langcode_entity'));
-      $url = Url::fromRoute("entity.$entity_type.canonical", [$entity_type => $entity_id], ['language' => $language]);
+      $url = Url::fromRoute("entity.$entity_type.canonical", [$entity_type => $entity_id]);
       $form_state->setRedirectUrl($url);
     }
   }
